src/AddPopUp.js

```diff
@@ -0,0 +1,42 @@
+import './AddPopUp.css'
+import {AppModes} from "./App";
+import {useState} from "react";
+
+function AddPopUp(props) {
+    const [value, setValue] = useState('');
```

Comment:
Does this represent the taskName that is to be added? Use a self-descriptive name rather than the generic value.